snippets/store.cson

```diff
@@ -0,0 +1,124 @@
+'.source.coffee':
+
+
+
+
+
+  'Ember Snippets: store':
+    'prefix': 'store--help'
+    'body':
+      """
+      ${1:
+        #  ____________________________________________________________________
+        # | use shorthand ->   or long version    ->   to generate these       |
+        # |  -----------  ->   ----------------   ->   ------------------      |
+        # |   sto.cre     ->  store.createRecord  ->   store.createRecord(...  |
+        # |   sto.del     ->  store.deleteRecord  ->   store.deleteRecord(...  |
+        # |   sto.fet     ->  store.fetch         ->   store.fetch(...         |
+        # |   sto.fil     ->  store.filter        ->   store.filter(...        |
+        # |   sto.pus     ->  store.push          ->   store.push(...          |
+        # |   sto.pusM    ->  store.pushMany      ->   store.pushMany(...      |
+        # |   sto.all     ->  store.all           ->   store.all(...           |
+        # |   sto.get     ->  store.getById       ->   store.getById(...       |
+        # |   sto.fin     ->  store.find          ->   store.find(...          |
+        # |____________________________________________________________________|
+      }
+      """
+
+
+
+
+  '1 Create Record in Store':
+    'prefix': 'store.createRecord'
+    'body':
+       """
+        store.createRecord('${1:property}',
```

Comment:
I only see one `createRecord()`. It doesn't matter what you type before the dot it should work both ways by default `store.createRecord()` == `model_instance.createRecord()` if the prefix is just `createRecord()`